server/db/index.js

```diff
@@ -1,14 +1,31 @@
 //this is the access point for all things database related!
 
-const db = require('./db')
+const db = require('./db');
+const User = require('./models/User');
+const Product = require('./models/Product');
+const Review = require('./models/Review');
+const Order = require('./models/Order');
+const Category = require('./models/Category');
 
-const User = require('./models/User')
+User.hasMany(Order);
```

Comment:
These all look good. I'd like to see the association between Orders and Products next, as well as thinking about how you can create an association there that can also serve as your shopping cart (you may even already be set up for this in your models)